sdk/scala-sdk/src/main/resources/reference.conf

```diff
@@ -0,0 +1,6 @@
+// for all the remaining configs check the java-sdk counterpart
+kalix {
+  library {
+    name = "kalix-scala-sdk"
+  }
+}
```

Comment:
a `reference.conf` cannot safely override another `reference.conf` the order they are loaded depends on the classpath, to do it like this it would have to be in an `application.conf` which is always loaded _after_ the reference conf.

I think this _may_ be easier to do with code from the code, where the Scala `Kalix` class creates the Java one.

Reply:
I see, that makes sense. I'll have a look at that option. Thanks Johan